gpustack/k8s/manifests.jinja

```diff
@@ -14,6 +14,60 @@ type: Opaque
 data:
   token: {{ config.token | b64encode }}
 ---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: pod-cm-creator
+rules:
+- apiGroups: [""]
+  resources: ["pods", "configmaps"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
```

Comment:
please append the RBAC support for the following resources.
- "services" -> *
- "pod/logs" -> get / watch
- "pods/exec" -> create

Reply:
Updated